br/pkg/lightning/config/config.go

```diff
@@ -511,16 +509,17 @@ type FileRouteRule struct {
 }
 
 type TikvImporter struct {
-	Addr               string   `toml:"addr" json:"addr"`
-	Backend            string   `toml:"backend" json:"backend"`
-	OnDuplicate        string   `toml:"on-duplicate" json:"on-duplicate"`
-	MaxKVPairs         int      `toml:"max-kv-pairs" json:"max-kv-pairs"`
-	SendKVPairs        int      `toml:"send-kv-pairs" json:"send-kv-pairs"`
-	RegionSplitSize    ByteSize `toml:"region-split-size" json:"region-split-size"`
-	SortedKVDir        string   `toml:"sorted-kv-dir" json:"sorted-kv-dir"`
-	DiskQuota          ByteSize `toml:"disk-quota" json:"disk-quota"`
-	RangeConcurrency   int      `toml:"range-concurrency" json:"range-concurrency"`
-	DuplicateDetection bool     `toml:"duplicate-detection" json:"duplicate-detection"`
+	Addr             string   `toml:"addr" json:"addr"`
+	Backend          string   `toml:"backend" json:"backend"`
+	OnDuplicate      string   `toml:"on-duplicate" json:"on-duplicate"`
+	MaxKVPairs       int      `toml:"max-kv-pairs" json:"max-kv-pairs"`
+	SendKVPairs      int      `toml:"send-kv-pairs" json:"send-kv-pairs"`
+	RegionSplitSize  ByteSize `toml:"region-split-size" json:"region-split-size"`
+	SortedKVDir      string   `toml:"sorted-kv-dir" json:"sorted-kv-dir"`
+	DiskQuota        ByteSize `toml:"disk-quota" json:"disk-quota"`
+	RangeConcurrency int      `toml:"range-concurrency" json:"range-concurrency"`
+	// Deprecated: only used to keep the compatibility.
+	DuplicateDetection bool `toml:"duplicate-detection" json:"duplicate-detection"`
```

Comment:
```suggestion
	DuplicateDetection bool `toml:"duplicate-detection" json:"-"`
```

do we really need to keep it? has this been published in previous versions?

Reply:
It exists in previous versions but we have never published it yet. I'm not sure if there is someone who is relying on this.

Reply:
if someone is relying on this after this PR it is broken since the code totally doesn't use the `DuplicateDetection` config at all.

Reply:
how about remove it?

Reply:
:shipit: